src/05-invoicing/invoices.apib

```diff
@@ -20,6 +20,8 @@ Get a list of invoices.
                     + draft
                     + outstanding
                     + matched
+            + date_before: `2016-02-04` (string, optional)
+            + date_after: `2016-02-04` (string, optional)
```

Comment:
Are the dates inclusive or exclusive? I think we used `_from` and `_until` for that reason on the internal deals.report endpoint.